broker/broker.go

```diff
@@ -119,27 +119,36 @@ func (b *CdnServiceBroker) LastOperation(
 
 	switch route.State {
 	case models.Provisioning:
+		instructions, err := b.manager.GetDNSInstructions(route.ChallengeJSON)
+		if err != nil {
+			return brokerapi.LastOperation{}, err
+		}
+		if len(instructions) != len(route.GetDomains()) {
+			return brokerapi.LastOperation{}, fmt.Errorf("Expected to find %d tokens; found %d", len(route.GetDomains()), len(instructions))
```

Comment:
@jmcarp I managed to get the broker into this state by doing the following:

* Create an instance with a single domain and complete the entire process
* Update the instance adding a second domain.

When this happens, the user is given no information:
```
Last Operation
Status: update in progress
Message: 
```
 and the broker reports this error in it's logs each time the CC polls it for an update:

```
2017-04-25T09:35:29.66-0700 [APP/PROC/WEB/0]ERR {"timestamp":"1493138129.649512053","source":"cdn-service-broker","message":"cdn-service-broker.lastOperation.unknown-error","log_level":2,"data":{"error":"Expected to find 2 tokens; found 1","instance-id":"1a8bf9bf-7092-4807-a008-05e7aa6f834e","session":"93"}}
```